plugins/dbviewer/frontend/public/javascripts/countly.views.js

```diff
@@ -36,34 +79,53 @@ window.DBViewerView = countlyView.extend({
         else {
             this.renderMain();
         }
+        $('body').off('click', '.collection-list-item').on('click', '.collection-list-item', function() {
+            app.navigate('#/manage/db/' + $(this).data('db') + '/' + $(this).data('collection'), false);
+            self.db = $(this).data('db');
+            self.collection = $(this).data('collection');
+            $('.dbviewer-documents-area').html("");
+            $('.dbviewer-documents-area').show();
+            $('.dbviewer-gray-area').show();
+            $('.dbviewer-left-side-1').html('<a href="javascript:void(0)" class="dbviewer-db-link-item dbviewer-gray" data-db="' + self.db + '">/ <strong>' + self.db + '</strong></a>');
+            $('.dbviewer-left-side-1').append('<a href="javascript:void(0)" class="dbviewer-collection-link-item dbviewer-gray" data-collection="' + self.collection + '"> /' + self.collection + '</a>');
+            self.executeQuery();
+        });
 
-        var prepareSwitch = function() {
-            // clear app-list
-            $('#app-list').html("");
-            // prepend "all apps" link to list
-            $('#app-list').prepend('<div data-value="all" class="app-option item" data-localize=""><span class="app-title-in-dropdown">' + $.i18n.map["common.all"] + '</span></div>');
-            // append list items
-            for (var key in countlyGlobal.apps) {
-                $('#app-list').append('<div data-value="' + countlyGlobal.apps[key]._id + '" class="app-option item" data-localize=""><span class="app-title-in-dropdown">' + countlyGlobal.apps[key].name + '</span></div>');
-            }
-            // set height 
-            if ($('#dbviewer').height() < (window.innerHeight - 150)) {
-                $('#dbviewer').css({"height": (window.innerHeight - 150) + "px"});
-                $('#accordion > div').css({"height": (window.innerHeight - 150) + "px"});
-            }
+        $('body').on('click', '.db-detail', function() {
+            self.db = $(this).data('db');
+            self.renderCollectionList();
+        });
 
-            if (store.get('dbviewer_selected_app')) {
-                $('#app-selector').html(store.get('dbviewer_selected_app').name);
+        if (window.location.hash.split('/').length === 4) {
```

Comment:
You cannot rely on `window.location.hash` because depending on server settings, sometimes it can include app_id, sometimes it won't, so the paths will differ

Instead use `Backbone.history.getFragment`
https://github.com/Countly/countly-server/blob/master/frontend/express/public/javascripts/countly/countly.template.js#L3272

which returns hash fragment without app_id, always.